presto-tests/src/test/java/com/facebook/presto/execution/TestQueues.java

```diff
@@ -266,25 +267,39 @@ public void testResourceGroupManagerRejection()
     }
 
     @Test(timeOut = 240_000)
+    public void testClientTagsBasedSelection()
+            throws Exception
+    {
+        try (DistributedQueryRunner queryRunner = TpchQueryRunner.createQueryRunner(ImmutableMap.of(), ImmutableMap.of("experimental.resource-groups-enabled", "true"))) {
```

Comment:
In other unit tests in this class we use `TestQueryRunnerUtil::createQueryRunner()`, which installs the `tpch` catalog. If you don't have any reasons to use `TpchQueryRunner::createQueryRunner()` we can just use the other one to make the tests consistent with each other.